ci/concourse/pipelines/nightly-release-pipeline.yml

```diff
@@ -61,7 +61,7 @@ resource_types:
   type: docker-image
   source:
     repository: ljfranklin/terraform-resource
-    tag: latest
+    tag: 0.12.13
```

Comment:
What is our plan to ensure we don't let this get stale?

Reply:
The hope is we'll be off Concourse the time it gets stale. The current resource has breaking changes so we can't use it.